modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json

```diff
@@ -3,7 +3,7 @@
 		{
 			"DBType": "Boolean",
 			"businessType": "Boolean",
-			"externalReferenceCode": "Active__c",
+			"externalReferenceCode": "Acitve_Partner__c",
```

Comment:
```suggestion
			"externalReferenceCode": "Active_Partner__c",
```

Reply:
I requested this change to the salesforce admins, waiting their confirmation.